apps/web/src/pages/quick-start/components/QuickStartWrapper.tsx

```diff
@@ -93,25 +94,30 @@ export function QuickStartWrapper({
 }
 
 export function Faq() {
+  const segment = useSegment();
+
+  function handleOnClick() {
+    segment.track(OnBoardingAnalyticsEnum.CLICKED_FAQ);
+  }
+
   return (
     <Center
       data-test-id="go-back-button"
       inline
       style={{
-        cursor: 'pointer',
         marginTop: '75px',
       }}
     >
       <span style={{ color: colors.B60 }}>Got stuck? </span>
-      <a href={faqUrl} style={{ marginLeft: '5px', color: '#DD2476' }}>
+      <a href={faqUrl} style={{ marginLeft: '5px', color: '#DD2476' }} onClick={() => handleOnClick} target="_blank">
```

Comment:
I tested locally and it was weird to experience being redirected on the same tab, therefor I added `target="_blank"` which will redirect to a new tab